kwords.py

```diff
@@ -0,0 +1,32 @@
+import requests
+import json
+
+url = "http://0.0.0.0:5000/"
+
+def get_keywords(text):
+
+    payload = json.dumps({
+    "text": text,
+    "lang": "en",
+    "n": 10
```

Comment:
**Suggestion:** The 'url' variable is currently a global variable, which could lead to potential issues if this script is imported as a module. It would be better to move it inside the 'get_keywords' function to limit its scope.
```suggestion
def get_keywords(text):

    url = "http://0.0.0.0:5000/"

    payload = json.dumps({
    "text": text,
    "lang": "en",
    "n": 10
    })
```